code/src/components/form/Form.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+
+export const Form = () => {
+  const [input, setInput] = useState({});
+
+  const handleInputChange = e => {
+    const target = e.currentTarget;
+
+    setInput({
+      ...input,
+      [target.name]: target.value
+    });
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log(input);
+  };
+
+  const handleResetForm = () => {
+    setInput({});
+  };
+
+  return (
+    <div>
+      <p>First name: {input.firstName}</p>
+      <p>First name: {input.lastName}</p>
+
+      <form onSubmit={handleSubmit}>
+        <label>First name:</label>
+        <input
+          name="firstName"
+          value={input.firstName || ''}
+          type="text"
+          onChange={handleInputChange}
+        />
+        <br />
+        <label>Last name:</label>
+        <input
+          name="lastName"
+          value={input.lastName || ''}
+          type="text"
+          onChange={handleInputChange}
+        />
+        <br />
+        <label>Web framework:</label>
+        <select
+          name="webFramework"
+          value={input.webFramework || ''}
+          onChange={handleInputChange}
+        >
+          <option value="jQuery">jQuery</option>
+          <option value="React.js">React.js</option>
+          <option value="Angular/Angular.js">Angular/Angular.js</option>
+          <option value="ASP.NET">ASP.NET</option>
+          <option value="Express">Express</option>
+          <option value="Spring">Spring</option>
+          <option value="Vue.js">Vue.js</option>
+          <option value="Django">Django</option>
+          <option value="Flask">Flask</option>
+          <option value="Laravel">Laravel</option>
+          <option value="Ruby on Rails">Ruby on Rails</option>
+          <option value="Drupal">Drupal</option>
```

Comment:
For this to be less code you can add the values as an array and then use `map()` to get the different `<option>`. You can read about how to do it here: https://reactjs.org/docs/lists-and-keys.html